handbook/ce/team-culture.md

```diff
@@ -0,0 +1,180 @@
+# Customer Engineering Team Culture
+
+CEs interact with most every other team at Sourcegraph. This page details those intersections as well as our team-specific operting procedures.
+
+
+* [How we interesect other teams](#how-we-intersect-other-teams)
+* [Recurring Team Meetings](#recurring-team-meetings)
+* [Out of Office Protocols](#ooo-protocols)
+* [Slack Channels](#ce-specific-slack-channels)
+* [CE Forward-Looking Vision](#forward-vision)
+
+--------
+
+## How we intersect other teams
+We work alongside just about every team at Sourcegraph. Here is how we add value to each other and a collaboration overview for each pairing.
+
+### Sales
+- **How CE adds value to sales:** CEs are the right-hand team for Sales by serving as technical experts; CEs lead product demos, guide customers through trials/POCs, lead production deployments, focuses on product usage and adoption.
```

Comment:
Add: builds technical champions

Reply:
Where do you suggest this be added? Under how we add value to sales?